.github/workflows/npm-publish-from-pr.yml

```diff
@@ -0,0 +1,73 @@
+name: npm publish from PR
+
+on:
+  workflow_dispatch:
+    inputs:
+      repo:
+        description: 'gh repo, like `scope/name`'
```

Comment:
can this be inferred?

Reply:
Yes, if rely on PR ref, we can obtain both repo and branch attrs